src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileMetadataService.scala

```diff
@@ -171,6 +171,7 @@ object FileMetadataService {
 
   val SHA256ClientSideChecksum = "SHA256ClientSideChecksum"
```

Comment:
Realise its not your code, but could you add some unit tests to check all these hardcoded values. They are pretty crucial, and some test would be good to at least to make devs think before changing them.

Reply:
I've added a test in to check this value. The only place it's being used is checking the status of the checksums which is pretty worthless not we're not updating one file at a time but I think I've changed enough for one PR.